src/templates/layouts/pressRelease/index.tsx

```diff
@@ -0,0 +1,155 @@
+import { PressRelease as FormattedPressRelease } from '@/types/formatted/pressRelease'
+import { LovellStaticPropsResource } from '@/lib/drupal/lovell/types'
+import { LovellSwitcher } from '@/templates/components/lovellSwitcher'
+import { formatDate } from '@/lib/utils/helpers'
+import { recordEvent } from '@/lib/analytics/recordEvent'
+import { ContentFooter } from '@/templates/common/contentFooter'
+
+export const PressRelease = ({
+  title,
+  releaseDate,
+  pdfVersion,
+  introText,
+  address,
+  fullText,
+  contacts,
+  downloads,
+
+  /*listing,*/
+  lovellVariant,
+  lovellSwitchPath,
+}: LovellStaticPropsResource<FormattedPressRelease>) => {
+  return (
+    <>
+      <div>
+        <main className="va-l-detail-page va-facility-page">
```

Comment:
can this switch to a `div`? pages already have a `<main>` element coming from `[[...slug]].tsx`. May also be able to lose the additional `<> <div>` wrappers.

Reply:
I saw this PR merged but this was never updated. This will cause a11y errors on all press release pages.